Sources/SearchableCollectionViewController/SearchableCollectionController.swift

```diff
@@ -1,8 +1,14 @@
 import Foundation
 import UIKit
 
+let SearchBarTag = 123
+
 open class SearchableCollectionController: UIViewController {
 
+    open var searchBar: UISearchBar {
```

Comment:
@3lvis an idea of how to expose it. Could be way better tho, but since it's not an accessible property from the collection view controller, I'm not sure what the best way to fetch it would be. I guess that's what tags are for.